test/k8s/datapath_configuration.go

```diff
@@ -665,11 +655,6 @@ var _ = Describe("K8sDatapathConfig", func() {
 				options["gke.enabled"] = "false"
 				options["tunnelProtocol"] = "vxlan"
 			}
-			if helpers.RunsWithKubeProxyReplacement() {
-				// BPF IPv6 masquerade not currently supported with host firewall - GH-26074
-				options["enableIPv6Masquerade"] = "false"
-			}
-			deploymentManager.DeployCilium(options, DeployCiliumOptionsAndDNS)
 			deploymentManager.DeployCilium(options, DeployCiliumOptionsAndDNS)
```

Comment:
Oh nice - this duplication was just accidental, right? Maybe worth backporting to v1.14, if it helps speeding up CI.

(some for the occurrences below)

Reply:
Yes it's likely some copy-paste mistake I did at the time, I can't find a single good reason to have this duplicated.

And yep, I plan to fix it on stable branches too.

Reply:
#31520 (v1.15), #31521 (v1.14)